README.md

```diff
@@ -1,5 +1,7 @@
 # IOTA-PHP
 
+[![Build Status](https://travis-ci.org/Techworker/iota-php.svg?branch=master)](https://travis-ci.org/Techworker/iota-php)
```

Comment:
Runs under my username? Are you sure?

Reply:
No, do you have the correct url ? :=)

Reply:
It looks like using the this url: `httpS://travis.org//iota-php/client`.